sequencer/Sequencer.py

```diff
@@ -235,7 +234,9 @@ def _build_edge_wkt(self):
         r = self.results
         # Iterate through the nodes and their parent
         for rank, fnode, tnode in zip(r.index, r['Sequence..Upstream.id'], r['Sequence..Vertex.id']):
-            if not np.isnan(fnode):
+            if fnode:
```

Comment:
Woah!!!! This is `False` if `fnode` is `0`!!!!!!! You should check for `None` explicitly.